components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -1531,7 +1531,6 @@ document.observe('xwiki:dom:loaded', function() {
     _updateDataRow : function(row) {
       // Generate the query string to pass to the measurement evaluation service
       var qs = this._createRowQueryString(row);
-      if (qs == '') {return;}
```

Comment:
I don't think that removing this line is optimal. Instead of removing it, which would cause a lot of extra work to be performed, there should be some extra cleanup before returning.